share/functions/fish_md5.fish

```diff
@@ -1,7 +1,7 @@
 function fish_md5
     if type -q md5sum
         # GNU systems
-        echo (md5sum $argv[1] | string split ' ')[1]
+        echo (echo $argv[1] | md5sum | string split ' ')[1]
```

Comment:
>So I should make a switch for fish_md5, where default passes as argument, and if switch is set, pipe it in?
But what's the equivalent for md5?

Yes.

> But what's the equivalent for md5?

According to http://www.unix.com/man-page/freebsd/1/MD5/ (googled for "bsd man md5"), it's `md5 -s STRING`.